export async function Register() {
    try {
        console.log('You are trying to register');
    }
    catch (error) {
        assert.fail(`There was some error: ${error}`)
    }
};

